HKMP/Game/Client/Entity/WanderingHusk.cs

```diff
@@ -0,0 +1,145 @@
+using System.Collections;
```

Comment:
I believe both this and the using directive below it are not used.

Reply:
My mistake, they were both unused. This should now be resolved.